agent/05_agent_configure.sh

```diff
@@ -245,6 +245,14 @@ function generate_cluster_manifests() {
     get_mirror_info
   fi
 
+  # Set BMC info
+  nodes_bmc_usernames=$(printf '%s,' "${AGENT_NODES_BMC_USERNAMES[@]}")
+  export AGENT_NODES_BMC_USERNAMES_STR=${nodes_bmc_usernames::-1}
+  nodes_bmc_passwords=$(printf '%s,' "${AGENT_NODES_BMC_PASSWORDS[@]}")
+  export AGENT_NODES_BMC_PASSWORDS_STR=${nodes_bmc_passwords::-1}
+  nodes_bmc_addresses=$(printf '%s,' "${AGENT_NODES_BMC_ADDRESSES[@]}")
+  export AGENT_NODES_BMC_ADDRESSES_STR=${nodes_bmc_addresses::-1}
+
```

Comment:
I saw these values get logged. Not a big issue, but unless strictly required I'd wrap them in a `set +x` just to be on the safe side

Reply:
Changed to wrap with `set +x`